src/App.tsx

```diff
@@ -1,93 +1,177 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect, useState, useContext, useMemo,
+} from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { getTodos, removeTodo, updateTodo } from './api/todos';
+import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const user = useContext(AuthContext);
+
+  const todosFromLocalStorage = localStorage.getItem('todos');
+  const initialTodos = todosFromLocalStorage
+    ? JSON.parse(todosFromLocalStorage)
+    : [];
+
+  const [todos, setTodos] = useState<Todo[]>(initialTodos);
+  const [isError, setIsError] = useState(false);
```

Comment:
You dont need to create a state for this. You can just check the length of the error message, if length = 0, there are no errors, if more then 0 - there are some errors.